### ISSUE TITLE
Training for IMDB - Running out of patience

### ISSUE THREAD

**Issue by monkeysforever** [allenai/vampire#64]
I am trying to train for the IMDB dataset, but only after a few epochs the training ends with the message Ran out of patience, which i guess is from early stopping. Can you recommend any hyperparamters I need to adjust to improve training?

**Comment by kernelmachine** [comment]
I assume you are talking about training VAMPIRE. I would try adjusting the learning rate, vocab size, hidden size of the model. Learning rate will probably have the biggest impact on how long you train for.

**Comment by monkeysforever** [comment]
thanks
